business/services.go

```diff
@@ -327,7 +327,7 @@ func (in *SvcService) GetService(namespace, service, interval string, queryTime
 	go func() {
 		// Maybe a future jaeger business layer
 		defer wg.Done()
-		eTraces, err = getErrorTracesFromJaeger(namespace, service, requestToken)
+		eTraces, err = in.businessLayer.Jaeger.jaeger.GetErrorTraces(namespace, service)
```

Comment:
I find a bit odd writing `in.businessLayer.Jaeger.jaeger...`
As I understand, the first `Jaeger` is the Jaeger service and the second `jaeger` is the jaeger client?
I would say, the function `GetErrorTraces` should be promoted into the service, so that another service doesn't access (pseudo-)private members of the Jaeger service (ie. the client). No?

Reply:
It's not working without that Jaeger I don't know why I need to check it

Reply:
@aljesusg I think it's because you didn't define `GetErrorTraces` in the business service

Reply:
sorry but I don't understand why

```go
business/services.go:330:41: in.businessLayer.Jaeger.GetErrorTraces undefined (type JaegerService has no field or method GetErrorTraces)
```

Reply:
the point is that it's defined in the client
```go
// ClientInterface for mocks (only mocked function are necessary here)
type ClientInterface interface {
	GetJaegerServices() (services []string, code int, err error)
	GetTraces(namespace string, service string, rawQuery string) (traces []*jaegerModels.Trace, code int, err error)
	GetTraceDetail(traceId string) (trace []*jaegerModels.Trace, code int, err error)
	GetErrorTraces(ns string, srv string) (errorTraces int, err error)
}
```
and I can set GetJaegerServices, GetTraces and GetTraceDetail but not GetErrorTraces

Reply:
Look at your file `business/jaeger.go` : you didn't define that function

Reply:
ooooo you're right

Reply:
pushed, thanks @jotak